composer.json

```diff
@@ -119,7 +119,7 @@
     "symfony/validator": "^6.4",
     "symfony/var-dumper": "^6.4",
     "symfony/web-profiler-bundle": "^6.4",
-    "symfony/webpack-encore-bundle": "^1.7",
+    "symfony/webpack-encore-bundle": "^1.7 || ^2.0",
```

Comment:
`symfony/webpack-encore-bundle < 1.13.2` can't be installed. So it makes no sense to set it lower.
```suggestion
    "symfony/webpack-encore-bundle": "^1.13.2 || ^2.0",
```

Reply:
It was probably a typo.

```suggestion
    "symfony/webpack-encore-bundle": "^1.17 || ^2.0",
```